browser/resources/settings/shortcuts_page/components/Keys.tsx

```diff
@@ -60,7 +60,8 @@ const keySymbols = {
   'Meta': <Padded>⌘</Padded>,
   'Alt': <Padded>⌥</Padded>,
   'AltGr': <Padded>⌥</Padded>,
-  'Control': <Padded>⌃</Padded>
+  'Control': <Padded>⌃</Padded>,
+  ' ': 'Space'
```

Comment:
Just curious - don't need translation?

Reply:
At the moment, we aren't translating any of the key names - for example `ArrowLeft` will show for the left arrow. Hopefully we revisit this soon :laughing: